webpack/assets/javascripts/react_app/components/AutoComplete/AutoCompleteSelectors.js

```diff
@@ -0,0 +1,23 @@
+export const selectAutocompleteError = (autocomplete, id) =>
```

Comment:
Today we understand that selectors should get the full state object rather than there own scope.
It allows us to reuse those selectors in different connected components.

Our current convention is to use a base-selector this way:
```js
export const selectAutocomplete = state => state.autocomplete;

export const selectAutocompleteError = (state, id) => selectAutocomplete(state)[id].error;
```